python/cogs/utils/converters.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from discord.ext.commands import BadArgument, Converter
+
+ALLOWED_CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ!?'`-"
+UNICODE_CHARACTERS = '𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹ǃ？’’-'
+
+
+class OffTopicName(Converter):
+    """A converter that ensures an added off-topic name is valid."""
+
+    @classmethod
```

Comment:
why is this a classmethod?

Reply:
Right, it was using the class attribute `ALLOWED_CHARACTERS` earlier, but not i shifted that to a constant, which is unnecessary, will fix it! Thanks.